app/scripts/templates/marketing_snippet.mustache

```diff
@@ -16,7 +16,7 @@
           <br>
           {{#t}}Download Firefox for Android.{{/t}}
         </p>
-        <a class="marketing-link marketing-link-android" href="{{ downloadLinkAndroid }}" data-marketing-id="{{ marketingId }}" data-marketing-type="android"><img width="152" height="45" src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
+        <a target="_blank" class="marketing-link marketing-link-android" href="{{ downloadLinkAndroid }}" data-marketing-id="{{ marketingId }}" data-marketing-type="android"><img width="152" height="45" src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
```

Comment:
Tangential question: the `alt` text for iOS finishes with an exclamation mark but the `alt` text for Android finishes with a period. Is that deliberate? Seems funny.